core-primitives/attestation-handler/Cargo.toml

```diff
@@ -17,7 +17,7 @@ log = { version = "0.4", default-features = false }
 base64 = { version = "0.13", features = ["alloc"], optional = true }
 chrono = { version = "0.4.19", features = ["alloc"], optional = true }
 rustls = { version = "0.19", optional = true }
-serde_json = { version = "1.0", optional = true }
+serde_json = { version = "1.0", features = ["preserve_order"], optional = true }
```

Comment:
Interesting, so this enables the feature globally? Well, not that is matters, I believe. It does already order the json keys according to the rust struct, so I doubt that we ever run into problems. 

And it would be stupid that it throws an error, when we try to create a rust struct based on a json with disordered fields.

Reply:
Should the feature be added to `serde_json_sgx` as well?

Reply:
Good point! This could have been the cause of an ugly bug!

Reply:
You are right! I fixed it now.